src/assets/scss/components/_signUpActivationModal.scss

```diff
@@ -84,4 +84,9 @@ sign-up-activation-modal {
       max-width: 300px;
     }
   }
+
+  hr,
+  .bottom-links {
+    display: none;
+  }
```

Comment:
I wonder if you could remove the <hr> or button links from the HTML instead of using CSS to remove them.